playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -73,6 +73,15 @@ EDXAPP_MONGO_PORT: 27017
 EDXAPP_MONGO_USER: 'edxapp'
 EDXAPP_MONGO_DB_NAME: 'edxapp'
 EDXAPP_MONGO_USE_SSL: False
+EDXAPP_MONGO_SSL_CLIENT_KEY: ""
+EDXAPP_MONGO_SSL_CLIENT_CERT: ""
+EDXAPP_MONGO_SSL_CA_CERT: ""
+EDXAPP_MONGO_SSL_CLIENT_CERT_PATH: !!null
+EDXAPP_MONGO_SSL_CA_CERT_PATH: !!null
+
+# When `EDXAPP_MONGO_USE_SSL: True`, set
```

Comment:
A note for posterity: I tried to set defaults for these based on the value of `EDXAPP_MONGO_USE_SSL`, but [`!!null` is broken](https://github.com/ansible/ansible/issues/17159) in Ansible. Something like `EDXAPP_MONGO_SSL_CLIENT_CERT_PATH: "{{ EDXAPP_MONGO_USE_SSL | ternary(EDXAPP_CFG_DIR ~ '/mongodb.pem', !!null) }}"` would have been amazing, but Ansible converts `!!null` to an empty string in many places where it shouldn't.